src/Microsoft.AspNetCore.Identity.EntityFrameworkCore/UserStore.cs

```diff
@@ -1196,5 +1197,69 @@ where userrole.RoleId.Equals(role.Id)
             }
             return new List<TUser>();
         }
+
+        private Task<IdentityUserToken<TKey>> FindToken(TUser user, string loginProvider, string name, CancellationToken cancellationToken)
+        {
+            var userId = user.Id;
+            return UserTokens.SingleOrDefaultAsync(l => l.UserId.Equals(userId) && l.LoginProvider == loginProvider && l.TokenName == name, cancellationToken);
+        }
+
+        // <inheritdoc>
+        public virtual async Task SetTokenAsync(TUser user, string loginProvider, string name, string value, CancellationToken cancellationToken)
```

Comment:
Have another name in mind @PinpointTownes 

We are calling it AuthenticationTokenStore currently, so Get/SetAuthToken matches